src/servers/src/http/prom_store.rs

```diff
@@ -89,7 +89,37 @@ pub async fn route_write_without_metric_engine(
         .start_timer();
 
     let is_zstd = content_encoding.contains(VM_ENCODING);
-    let (request, samples) = decode_remote_write_request(is_zstd, body).await?;
+    let (request, samples) = decode_remote_write_request(is_zstd, body, true).await?;
+    // reject if physical table is specified when metric engine is disabled
+    if params.physical_table.is_some() {
+        return UnexpectedPhysicalTableSnafu {}.fail();
+    }
+
+    let output = handler.write(request, query_ctx, false).await?;
+    crate::metrics::PROM_STORE_REMOTE_WRITE_SAMPLES.inc_by(samples as u64);
+    Ok((
+        StatusCode::NO_CONTENT,
+        write_cost_header_map(output.meta.cost),
+    )
+        .into_response())
+}
+
+/// Same with [remote_write] but won't store data to metric engine.
+#[axum_macros::debug_handler]
+pub async fn route_write_without_metric_engine_and_strict_mode(
```

Comment:
maybe it makes sense to add more docs on the "mode" of this method by explicitly define `metric_engine` `strict_mode` values which are assumed for this method, wdyt?